modules/45_carbonprice/functionalForm/postsolve.gms

```diff
@@ -363,6 +364,17 @@ loop(regi,
       * rPower( (t.val - p45_interpolation_startYr(regi)) / (p45_interpolation_endYr(regi) - p45_interpolation_startYr(regi)), p45_interpolation_exponent(regi));
   pm_taxCO2eq(t,regi)$(t.val ge p45_interpolation_endYr(regi)) = p45_taxCO2eq_regiDiff(t,regi);
 );
+$else.taxCO2startYearValue4
```

Comment:
Same as in datainput - recreating correct interpolation after adjustment of `p45_taxCO2eq_anchor`